package.json

```diff
@@ -11,7 +11,7 @@
     "test": "npm run lint"
   },
   "dependencies": {
-    "@aeternity/aepp-components": "2.0.11",
+    "@aeternity/aepp-components": "^2.0.13",
```

Comment:
Usually, the components package has a break changes in a patch releases, so please remove `^` to match with the particular version of components package.